Cargo.toml

```diff
@@ -16,7 +16,7 @@ authors = [
     "Andrés Medina <andres.medina@redcanary.com>",
     "Vince Bundage <vince.bundage@redcanary.com>"
 ]
-edition = "2021"
+edition = "2018"
```

Comment:
I think we have some 2021-isms that are causing this to fail CI. Is 2018 necessary here?

Specifically, the `thread-priority` tests are failing with:
```
no method named `try_into` found for type `u8` in the current scope
```

If I raise the edition back to 2021, everything passes.

Reply:
Oh wow, beat me to it.

Reply:
I fixed the issue by adding the import in a sneaky force push 😁